src/components/biggive-button/biggive-button.scss

```diff
@@ -96,66 +109,41 @@
     @include button-grey-dark();
 }
 
-.button-clear-primary {
-    @include button-clear-primary();
-}
-
-.button-clear-secondary {
-    @include button-clear-secondary();
-}
-
-.button-clear-tertiary {
-    @include button-clear-tertiary();
-}
-
-.button-clear-brand-1 {
-    @include button-clear-brand-1();
-}
-
-.button-clear-brand-2 {
-    @include button-clear-brand-2();
-}
-
-.button-clear-brand-3 {
-    @include button-clear-brand-3();
-}
-
-.button-clear-brand-4 {
-    @include button-clear-brand-4();
-}
-
-.button-clear-brand-5 {
-    @include button-clear-brand-5();
-}
-
-.button-clear-brand-6 {
-    @include button-clear-brand-6();
-}
-
-.button-clear-white {
-   @include button-clear-white();
-}
-
-.button-clear-black {
-    @include button-clear-black();
- }
-
- .button-clear-grey-extra-light {
-    @include button-clear-grey-extra-light();
- }
- .button-clear-grey-light {
-    @include button-clear-grey-light();
- }
-
- .button-clear-grey-medium {
-    @include button-clear-grey-medium();
- }
-
- .button-clear-grey-dark {
-    @include button-clear-grey-dark();
- }
-
-
 .button-rounded {
     @include button-rounded();
  }
+
+ $colours: (
+  "primary": $colour-primary-blue,
+  "secondary": $colour-secondary-green,
+  "tertiary": $colour-tertiary-coral,
+  "brand-1": $colour-brand-cc-red,
+  "brand-2": $colour-brand-wgmf-purple,
+  "brand-3": $colour-brand-gmf-green,
+  "brand-4": $colour-brand-emf-yellow,
+  "brand-5": $colour-brand-c4c-orange,
+  //"brand-6": $colour-brand-grey,
+  "brand-afa-pink": $colour-brand-afa-pink,
+  "brand-mhf-turquoise": $colour-brand-mhf-turquoise,
+  "white": $colour-white,
+  "black": $colour-black,
+  "grey-extra-light": $colour-grey-extra-light,
+  "grey-light": $colour-grey-light,
+  "grey-medium": $colour-grey-medium,
+  "grey-dark": $colour-grey-dark
+  );
+
+ @each $brand, $colour in $colours {
+    .button-clear-#{$brand} {
+        color: $colour;
+        background-color: transparent;
+        text-decoration: underline;
+        border: 0;
+        padding-left: 0;
+        padding-right: 0;
+        &:hover {
```

Comment:
I looked through the deleted mixins, I see they're all the same except some include a rule to keep colour the same and some don't. Is there a reason not to have the hover colour set here? I think it's almost certainly not a deliberate distinction, probably it was just found to be needed on some of the more recently done ones.